test/tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
+    "target": "ES2017",
+    "module": "commonjs",
+    "outDir": "../",
+    "declaration": false,
+    "rootDir": "../",
+    "rootDirs": [
+      "../src",
+      "../types"
+    ],
+    "types": [
+      "node"
```

Comment:
It is common practice to add `mocha` here so you don't have to import functions like `describe` or `it` from mocha in every spec file.